js/navigation-search.js

```diff
@@ -47,7 +50,8 @@ bcpl.navigationSearch = (($) => {
 	 * Click event handler for the search button.
 	 */
 	const searchButtonClicked = (event) => {
-
+		const searchTerms = $(event.currentTarget).siblings('input').first().val();
+		window.location = `/dist/search.html?q=${searchTerms}&page=1&resultsPerPage=10`;
```

Comment:
I'm just wonder if url's should be specified in a configuration or some other way so that that can be easily changes, rather than hidden in a method